test/users-test.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const userData = data.testData;
```

Comment:
I think you need a line above this thats something like `const data = require('./data/users.js');`

Reply:
Will add this